### DISCUSSION TITLE
Gaming & Collusion Potential

### DISCUSSION THREAD

**Discussion by joesixpack** [subquery/subql#993] [Frontier TestNet - Bug Reporting]
There may be some gaming and collusion potential here, because a Delegator can delegate and/or undelegate in the current era before an increase and/or decrease in the Indexer's self-stake and/or commission has formally activated in the next era (or two).

![image](https://user-images.githubusercontent.com/23145642/167323608-f1ce55ac-bcf4-4035-81ed-f1261d8fa24e.png)

**Comment by mzxyz** [comment]
@lilian79511 @CaiYiLiang

**Comment by jamesbayly** [comment]
How might there be collusion? It's designed so that an Indexer can't change the commission rate unfairly before "advertising" it with their delegators
